swagger2.0/oic.sec.account.swagger.json

```diff
@@ -73,17 +73,19 @@
     "Account-request" : {
       "properties": {
         "authprovider": {
-          "description": "The name of Authorization Provider through which Access Token was obtained",
+          "description": "The name of Authorization Provider through which Access Token was obtained.",
+          "writeOnly": true,
           "type": "string"
         },
         "accesstoken" : {
-          "description": "Access-Token used for communication with OCF Cloud after account creation",
-          "pattern": "(?!$|\\s+).*",
+          "description": "Access Token used to authorize and associate the TLS connection for communication with the OCF Cloud with the Device ID, or the Authorization Code which is then verified and exchanged for the Access Token during Device Registration.",
+          "writeOnly": true,
           "type": "string"
         },
         "di": {
-          "description": "Format pattern according to IETF RFC 4122.",
+          "description": "Unique Device identifier. Format pattern according to IETF RFC 4122.",
           "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "writeOnly": true,
```

Comment:
Is "writeOnly" a valid Swagger keyword?  I've never seen it before.  This needs to be checked with the DMWG.

The schema for the CCC Resource does not use the "writeOnly" tag: https://github.com/openconnectivityfoundation/core-extensions/blob/master/swagger2.0/oic.r.coapcloudconf.swagger.json

Reply:
writeOnly is supported by swagger (https://swagger.io/docs/specification/data-models/data-types/) and it was by recommendation from DMWG that I specified Write Only access in resource tables

Reply:
I wonder why the CCC Resource doesn't use it???
In any event, this is a Resource hosted by the Cloud so it won't impact the CTT.

Reply:
CTT acts as a Cloud for d2c testing, so there might be some impact

Reply:
Now that you've changed back to readOnly, there should be zero impact to the CTT.  I think we're good.